modules/grafana/files/dashboards/sidekiq.json

```diff
@@ -273,7 +273,165 @@
     },
     {
       "collapse": false,
-      "height": 327,
+      "height": 408,
+      "panels": [
+        {
+          "aliasColors": {},
+          "bars": true,
+          "dashLength": 10,
+          "dashes": false,
+          "datasource": "Graphite",
+          "fill": 1,
+          "id": 10,
+          "legend": {
+            "alignAsTable": true,
+            "avg": true,
+            "current": true,
+            "max": true,
+            "min": false,
+            "show": true,
+            "total": false,
+            "values": true
+          },
+          "lines": false,
+          "linewidth": 1,
+          "links": [],
+          "nullPointMode": "null",
+          "percentage": false,
+          "pointradius": 5,
+          "points": false,
+          "renderer": "flot",
+          "seriesOverrides": [],
+          "spaceLength": 10,
+          "span": 6,
+          "stack": false,
+          "steppedLine": false,
+          "targets": [
+            {
+              "refId": "A",
+              "target": "aliasByNode(summarize(averageSeriesWithWildcards(stats.gauges.govuk.app.$Application.*.workers.queues.*.runtime, max, 5), '$Aggregation', 'max', false), 7)"
```

Comment:
I'm a bit confused by the second parameter to `averageSeriesWithWildcards` here. Max isn't really an average... I'm guessing this works though.

Reply:
You're right to be confused - it's not even a [validate parameter](https://graphite.readthedocs.io/en/latest/functions.html#graphite.render.functions.averageSeriesWithWildcards). Graphite seems to be interpreting it as a number, though 👏. Fixed to use `consolidateBy(max)`, like we do everywhere else.

Reply:
As far as I understand, `consolidateBy` doesn't fundamentally change what metrics you get back, but subtly changes the aggregations that Graphite implicitly does internally (which isn't very relevant here, as the aggregation and period is very explicit in these panels). I'm not saying that using it is a problem, just that I don't think using it fixes anything.

Reply:
You're right - sorry, was getting confused with the approach we took in 'Email Alert Api Metrics'. The function probably has some value, since even with the aggregation period Graphite may still need to downsample when drawing the graphs. I've replaced it with a `groupByNode` approach and updated all the panels to be consistent.

Does that make more sense?

Reply:
Yeah, I think so.